doc/cookbook/source/examples/kernel/sub_sequence_kernel.rst

```diff
@@ -0,0 +1,51 @@
+=========================
+String Subsequence Kernel
+=========================
+
+The String Subsequence Kernel (SSK) is discussed by :cite:`lodhi2002ssk`.
+
+A subsequence is any ordered sequence of :math:`n` characters occurring in
+the text, though not necessarily contiguous.
+
+More formally, string :math:`u` is a subsequence of string :math:`s`, :math:`iff` there
```

Comment:
Too much detail. The cookbook examples should only cover basics (in particular for kernels) i.e. only one equation or so. Preferably only use math symbols for things that are later used in the code.

"..... This kernel is an inner product in the feature space generated by all sub-sequences of length n. The sub-sequences are weighted by the exponentially decaying factor λ. More weight is given to those occurrences that are nearly contiguous. The implementation uses an efficient dynamic programming strategy."